bin/apisix

```diff
@@ -187,6 +187,20 @@ http {
             apisix.http_ssl_phase()
         }
 
+        location /dashboard {
```

Comment:
We'd better use `/apisix /dashboard`, we need to avoid conflicts with user rules.